src/ui/widgets/CategoryWidgetUI.js

```diff
@@ -339,9 +339,9 @@ function CategoryWidgetUI(props) {
       drawFrame: (val) => setAnimValues(val),
       requestRef
     });
-
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
In order to avoid the 'delayed animation' we were talking:
`prevAnimValues` in the dependencies array causes the delay, I recommend to move the animation logic to an `useCallback`, something like:
```javascript
const launchAnimation = useCallback(() => {
   ...
}, [data, prevValue, setValue]);

useEffect(() => {
  launchAnimation();
  // eslint-disable-next-line react-hooks/exhaustive-deps
  return () => cancelAnimationFrame(requestRef.current);
}, [data, setValue, launchAnimation]);
```
We still need to check that disabled line!